src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -659,4 +659,57 @@ class JavaMigrations {
       .insert()
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "14.ea.14-open", _))
   }
+
+  @ChangeSet(order = "190", id = "190-add_12.0.2.hs-adpt", author = "vpavic")
+  def migrate190(implicit db: MongoDatabase): Unit = {
+    val candidate = "java"
+    val oldVersion = "12.0.1.hs-adpt"
+    val newVersion = "12.0.2.hs-adpt"
+    Seq(Linux64, MacOSX, Windows).foreach(removeVersion(candidate, oldVersion, _))
+    List(
+      Version(candidate, newVersion, "https://github.com/AdoptOpenJDK/openjdk12-binaries/releases/download/jdk-12.0.2%2B10/OpenJDK12U-jdk_x64_linux_hotspot_12.0.2_10.tar.gz", Linux64, Some(OpenJDK)),
```

Comment:
The vendor should be `AdoptOpenJDK`, as `OpenJDK` refers to binaries found on java.net.

Reply:
Argh, that's a _regression_ due to c&p-ing back to `List` approach. Should be all good now.